docs/vi/index.md

```diff
@@ -0,0 +1,303 @@
+---
+layout: defaut
+Title: Học sâu
```

Comment:
`title:` not `Title:`. My script won't understand capital letters. It's case sensitive.

Reply:
I corrected this error.